### ISSUE TITLE
Request for Enhancements: Real-time Data and "Aktuellen Abdruck" Support

### ISSUE THREAD

**Issue by yachty66** [bundesAPI/handelsregister#22]
Hello,

I have recently developed a custom scraper for the Handelsregister and have come across your API. It appears to be a promising tool that could potentially streamline my operations. However, I have a couple of queries and potential enhancement requests that I would like to discuss.

Real-time Data: For my use case, it is crucial to have access to the most recent data. I would like to know if your API provides real-time or near real-time updates from the Handelsregister.

"Aktuellen Abdruck" Support: In addition to the data, I also require the "Aktuellen Abdruck", a PDF document that can be downloaded from the respective website. I noticed that the current version of your API does not seem to support this feature. I would like to inquire if there are any plans to incorporate this functionality in the future, or if there are any workarounds available at the moment.

In summary, my questions are:

Does your API provide the most up-to-date data from the Handelsregister?
Is it possible to obtain the "Aktuellen Abdruck" using your API?

**Comment by yachty66** [comment]
This was a while ago. I am not working on the project anymore!

> still having the problem with "Aktuellen Abdruck"??

**Comment by mauhai** [comment]
@m-derra If you or others still need support here, I built a hosted API with which you can access the respective files. Its not ideal and still in beta, but should solve your problem. I'm hoping to be able to release it to the public in the future. Feel free to reach out to me via hai [at] openregisters [dot] com.
